session/bench_test.go

```diff
@@ -36,6 +37,10 @@ var smallCount = 100
 var bigCount = 10000
 
 func prepareBenchSession() (Session, *domain.Domain, kv.Storage) {
+	config.UpdateGlobal(func(cfg *config.Config) {
```

Comment:
Will session variable `TiDBEnableSlowLog` take effects? I think it is better than global config, if it works.

Reply:
I'm not sure is the CI stable currently, or is it affected by this change, let me re-run it.